src/Model/Nodeinfo.php

```diff
@@ -147,7 +139,7 @@ public static function getServices(): array
 		if (Addon::isEnabled('tumblr')) {
 			$services['outbound'][] = 'tumblr';
 		}
-		if (Addon::isEnabled('twitter') || Addon::isEnabled('buffer')) {
+		if (Addon::isEnabled('twitter')) {
```

Comment:
This method would be a great place for a hook.

Reply:
This whole functionality is some really ugly place of code.